arborist/user.go

```diff
@@ -164,7 +164,7 @@ func fetchUserPolicyInfo(db *sqlx.DB, user_name string, policy_name string) (*Us
 			FROM usr_policy
 			INNER JOIN policy ON policy.id = usr_policy.policy_id
 			INNER JOIN usr on usr_policy.usr_id = usr.id 
-			WHERE usr.name like $1 and policy.name like $2;
+			WHERE LOWER(usr.name) like $1 and policy.name like $2;
```

Comment:
should `user_name` on L173 be lowered?

Reply:
Addressed in this commit: https://github.com/uc-cdis/arborist/pull/165/commits/792db81455b841cea4226fa20e00a214278b26ce